src/hist/basehist.py

```diff
@@ -461,3 +461,26 @@ def plot_pie(
         import hist.plot
 
         return hist.plot.plot_pie(self, ax=ax, **kwargs)
+
+    def stack(self: T, *args: Union[int, str]) -> Any:  # todo: should return Stack here
```

Comment:
```suggestion
    def stack(self: T, *args: Union[int, str]) -> "hist.stacks.Stack":
```

And add an `import hist.stacks` at the top. You can avoid a circular import error by not referring to "Stack" directly outside of a function body.

Reply:
Fxied.